.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
     - WITH_SCIPY=true WITH_CODECOV=true
```

Comment:
Remove this, since it's not relevant for you. Just remove the whole matrix section

Reply:
And also remove the WITH_SCIPY variable from the 3.8-dev build